apps/src/templates/lessonOverview/activities/ProgressionDetails.jsx

```diff
@@ -25,6 +26,17 @@ export default class ProgressionDetails extends Component {
 
   handleBubbleClick = level => {
     this.setState({previewingLevel: level});
+    firehoseClient.putRecord(
+      {
+        study: 'lesson-plan',
+        study_group: 'teacher-lesson-plan',
+        event: 'click-level',
```

Comment:
nit: should this be something like `click-level-preview` to make it clear that this isn't clicking into the level?